apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -288,7 +287,12 @@ export const onSaveFail =
           if (body?.details?.profaneWords?.length > 0 && flaggedProperties) {
             errorMessage = `Profanity detected in the ${flaggedProperties} and cannot be updated. Please try again.`;
           }
-          dispatchSaveFailNotification(dispatch, errorMessage);
+          const includeInChatHistory = flaggedProperties ? true : false;
+          dispatchSaveFailNotification(
+            dispatch,
+            errorMessage,
+            !!includeInChatHistory
```

Comment:
nit: do you need the separate `includeInChatHistory` local variable? can this just be `!!flaggedProperties`?

Reply:
Sure!